CHANGELOG.md

```diff
@@ -9,6 +9,7 @@ and [PEP 440](https://www.python.org/dev/peps/pep-0440/).
 
 ## [2.5.1] - not released yet
 ### Added
+- new parameters `new_x` and `new_y` for `cell()` and `multi_cell()`, replacing `ln=0`, thanks to @gmischler
```

Comment:
This should be added to the `2.5.2` section above.
You should probably rebase your branch on this repo `master` branch

Reply:
I thought I had the most recent data... now I actually do!